requirements.txt

```diff
@@ -11,7 +11,7 @@ torchaudio
 torchtext
 torchvision
 pydantic<2.0
-transformers>=4.33.2
+transformers>=4.33.2,<4.35.0 # pinning since version 4.35.0 on 11/2/2023 causes IndexError in PEFT
```

Comment:
Is there an issue on transformers to track, so we can know when to unpin?

Reply:
@tgaddair No -- good catch; I created it: `https://github.com/huggingface/transformers/issues/27278`.  Thank you.